sdk/src/types/block/slot/commitment.rs

```diff
@@ -22,7 +22,7 @@ pub struct SlotCommitment {
     /// It is calculated based on genesis timestamp and the duration of a slot.
     index: SlotIndex,
     /// The commitment ID of the previous slot.
-    #[cfg_attr(feature = "serde", serde(rename = "prevId"))]
+    #[cfg_attr(feature = "serde", serde(rename = "previousCommitmentId"))]
```

Comment:
https://github.com/iotaledger/tips-draft/blob/tip46/tips/TIP-0046/tip-0046.md#slot-commitment

says "Prev Id"

Reply:
Maybe we should just rename the field and remove the serde tag?

Reply:
Idk, I don't like that they don't use slot here

Reply:
🤷‍♀️

Reply:
I see, thanks